docs/tutorials/enterprise_beginner/2_preparing_the_landing_page.md

```diff
@@ -161,23 +165,22 @@ landing_page:
         Identifier\ContentType: "landing_page"
 ```
 
-You can place this block anywhere before or after other view configuration blocks, but remember that the indentations must match and the block must be placed under the `full` key.
 
-!!! tip
+!!! note
 
-    The `views.yml` file already contains a handful of view config blocks, and the `views/full` folder has templates that correspond to them. They are used to render the existing content we have in our website - articles, dog breed information and tips. In a clean installation these configurations and the folder would not exist and you would have to build all view templates from scratch.
+    The `views.yml` file already contains a handful of view config blocks, and the `views/full` folder has templates that correspond to them. They are used to render the existing content you have in your website - articles, dog breed information and tips. In a clean installation these configurations and the folder would not exist and you would have to build all view templates from scratch.
 
-After adding this template you can check your new Landing Page. The part between menu and footer should be empty, which is the desired result – it should be empty, because we have not added any actual content to it.
+After adding this template you can check your new Landing Page. The part between menu and footer should be empty, because you have not added any actual content to it yet.
```

Comment:
We still have that information: The part between menu and footer should be empty, because you have not added any actual content to it yet.

"it's the desired result" was a repetition of "should be empty" for me